Sources/Mappo/main.swift

```diff
@@ -644,31 +661,39 @@ class State {
 		actions = [:]
 	}
 
-	func checkWin() async throws {
-		let aliveEvil = roles.filter { roles[$0.key] == .werewolf }
+	func checkWin(whoDied: Snowflake, why: DeathReason) -> Optional<(String, Team)> {
```

Comment:
```suggestion
	func checkWin(whoDied: Snowflake, why: DeathReason) -> (String, Team)? {
```

can we use a struct + a symbolic enum instead of returning a tuple?